plugin/latexlivepreview.vim

```diff
@@ -156,11 +162,12 @@ EEOOFF
                 \ fnamemodify(l:root_file, ':t:r') . '.pdf'
 
     let b:livepreview_buf_data['run_cmd'] =
-                \ 'env ' .
+                \ (has('win32') ? 'set' : 'env') . ' ' .
                 \       'TEXMFOUTPUT=' . l:tmp_root_dir . ' ' .
                 \       'TEXINPUTS=' . l:tmp_root_dir
                 \                    . ':' . b:livepreview_buf_data['root_dir']
                 \                    . ': ' .
+                \ (has('win32') ? '&& ' : '') .
```

Comment:
Could you explain this line?

Reply:
On *NIX, `env` command has the syntax `env [name=value]... [command [arg]...]`; but on Windows, an environment variable can only be set by the `set` command, whose syntax is `set [variable=[string]]`. So something is needed to combine the `set` command and the compile command. The only syntax to do this is `cmd1 || cmd2` or `cmd1 && cmd2` which behaves almost exactly the same as the same shell syntax. So I chose `&&`, the space after `&&` is to beautify the `run_cmd`.

Unfortunately, according to its syntax, `set` can only set one variable, and the `set` command should be explained as `set TEXMFOUTPUT="... TEXINPUTS=..."`(i.e. `TEXINPUTS=...` itself is a part or `TEXMFOUTPUT`'s value), but it works as `TEXINPUTS` is set properly... I need more work to find out the reason.

Sorry for such long explaination, I have tried my best to simplify and clearify it.